apps/passport-client/pages/index.tsx

```diff
@@ -160,10 +187,23 @@ function RouterImpl() {
           <Route path="add" element={<AddScreen />} />
           <Route path="prove" element={<ProveScreen />} />
           <Route path="scan" element={<ScanScreen />} />
-          <Route path="verify-zupass" element={<VerifyScreen />} />
           <Route
-            path="verify-devconnect"
-            element={<DevconnectCheckinScreen />}
+            path="verify"
+            element={
+              <VerifyScreen />
+            }
+          />
+          <Route
+            path="checkin"
```

Comment:
This takes a serialized PCD as a parameter, and works with the "full fat" QR code. `checkin-by-id` below takes the ticket ID as a parameter.